classlib/src/main/java/org/teavm/classlib/java/lang/TCharacter.java

```diff
@@ -177,7 +178,7 @@ public static boolean isSurrogatePair(char high, char low) {
     }
 
     public static int charCount(int codePoint) {
-        return isSupplementaryCodePoint(codePoint) ? 2 : 1;
+        return codePoint >= MIN_SUPPLEMENTARY_CODE_POINT ? 2 : 1;
```

Comment:
Why?

Reply:
Because charCount(Integer.MAX_VALUE) for instance is 2 (despite it's meaningless).